### ISSUE TITLE
Export to PDF

### ISSUE THREAD

**Issue by Josh1711** [lanternbox/exploreCRS#10]
I just tried on one candidates profile luciajin (3890) to export to PDF and this is a new issue unrelated to the previous one I mentioned in today's meeting. Once I clicked the export and it opened in Adobe I received the message as shown in the Screenshot below:

![image](https://cloud.githubusercontent.com/assets/23716044/20591022/63d82444-b261-11e6-8170-e181b637a674.png)

It said that there was an internal error but wouldn't explain more. This is different from the other issue and has only seemed to happen on her profile so I am not sure how to resolve. Thanks.

**Comment by Josh1711** [comment]
I will try to look into it myself but was just wondering why it happened just for the one profile.

**Comment by lantern-andrew** [comment]
Hi Josh, thanks for all the clear details and the screenshot.

Just now, I tried exporting a profile for this candidate and I was able to open it normally in the pdf viewer we use here (Adobe Acrobat Pro DC). I attached a screenshot and also the pdf I created.

I did a quick review of the data for this profile, and I didn't see anything too unusual that might be causing an issue.

The error you got from Adobe ("An internal error occurred") is not at all clear but I did a quick search - some users were able to get around this by changing their Adobe Acrobat settings: https://forums.adobe.com/message/4262880. Could I suggest that you give that a try? 

You may also try downloading the image, resaving it in an image editor, and uploading it again - just in case there is something about the file that is causing issues with your Acrobat version.

If it is still not working, there is not much else we can do to troubleshoot (as we can't recreate the error here), but if the attached pdf works for you, we could create more later if needed, on a case-by-case basis.

Thanks,
Andrew

<img width="1136" alt="screen shot 2016-11-24 at 5 48 35 pm" src="https://cloud.githubusercontent.com/assets/23451422/20593690/47cc5632-b26e-11e6-85ac-d85d8d4d06bd.png">

[profile.pdf](https://github.com/lanternbox/exploreCRS/files/610936/profile.pdf)

**Comment by Josh1711** [comment]
Hey Andrew, Thanks for the quick reply, I have also found a way that solved it too. What I did was removed her photo from the profile, cropped and resized it and changed the file format as for some reason she had uploaded as a a bitimage or something like that. I resaved as JPEG and then also shrunk the size and seemed to resolve the issue. I think many of the problems related to the Adobe and PDF's are exactly that, related to Adobe. I think it is more a problem with the software as you said. I will however, follow your info about changing the settings on Acrobat. 

Thanks again for your help.

Josh
